### PR TITLE
Genesis accounts & keypair config

### PR DESCRIPTION
**Pull request by dincho**
[PT 153171793](https://www.pivotaltracker.com/n/projects/2124891/stories/153171793)

This PR also introduce new ["tester" environment](https://github.com/aeternity/infrastructure/pull/10) where we'll deploy the UAT genesis account keypairs.

https://github.com/aeternity/infrastructure/pull/10 needs to be merged first.

### CODE REVIEW

**Comment by lucafavatella** [comment]
@dincho  This PR needs to start from new DB folder in UAT environment.

**Comment by dincho** [comment]
> As this changes genesis in UAT, please check that DB folder will be a fresh one.

Good point, I think it will be OK:
- Current UAT is [DB3](https://github.com/aeternity/epoch/blob/v0.3.1-big-spenders/deployment/ansible/deploy.yml#L12).
- Master is already at [DB6](https://github.com/aeternity/epoch/blob/master/deployment/ansible/deploy.yml#L12)

**Comment by lucafavatella** [comment]
@dincho I believe OSX jobs are not needed for this PR. I suggest that when you are ready to do so (infrastructure is up, Travis CI config is set) you merge.
Cc @sennui @happi

**Comment by lucafavatella** [comment]
And this looks like not conflicting with any other PR.

File: deployment/ansible/group_vars/uat/accounts.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+genesis_accounts:
+  "BKSRJ6RvBkUhW51L15s0v1LyhbLRZndE8RNAWy3BjonpcwZv+7yRYYQ567v7y2aRMwg1Jibue8fwe5SLX3ekArc=": 100000000000001
```
Comment (lucafavatella) [inline deployment/ansible/group_vars/uat/accounts.yml]: @dincho Will the users be able to connect to UAT without modifying the accounts.json in the released package?

Comment (dincho) [reply deployment/ansible/group_vars/uat/accounts.yml]: @lucafavatella see the changes in 112aa5b

File: deployment/ansible/deploy.yml [hunk 2]
```diff
@@ -5,11 +5,16 @@
 
   vars:
     project_root: "{{ ansible_env.HOME }}/node"
+    genesis_accounts_path: "{{ project_root }}/lib/aecore-0.1.0/priv/.genesis/accounts.json"
+
     env: unknown
     config:
       chain:
         persist: true
         db_path: "./db7"
```
Comment (lucafavatella) [inline deployment/ansible/deploy.yml]: I believe this shall be bumped.

File: deployment/ansible/templates/epoch.yaml.j2 [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 {% if 'epoch' in group_names %}
 peers:
-    {%for host in ansible_play_hosts | difference([inventory_hostname]) %}
+    {% for host in ansible_play_hosts | difference([inventory_hostname]) %}
 - "http://{{ hostvars[host]['ansible_default_ipv4']['address'] }}:3013/"
     {% endfor %}
 {%- endif %}
 
+keys:
+    dir: {{ config.keypair.dir }}
```
Comment (lucafavatella) [inline deployment/ansible/templates/epoch.yaml.j2]: I see this is "keys". I wonder if it shall be "./keys" or contain `{{ project_root }}`. Did you test locally?

Comment (lucafavatella) [reply deployment/ansible/templates/epoch.yaml.j2]: /me testing locally

Comment (dincho) [reply deployment/ansible/templates/epoch.yaml.j2]: Yes, I tested it locally. Looks like it's relative to the "epoch" root

Comment (lucafavatella) [reply deployment/ansible/templates/epoch.yaml.j2]: Local test passed. With "keys".

**Pull request merged**